pkg/netflow/portrollup/portrollup.go

```diff
@@ -137,9 +139,36 @@ func (prs *EndpointPairPortRollupStore) UseNewStoreAsCurrentStore() {
 	defer prs.mu.Unlock()
 
 	prs.curStore = prs.newStore
-	prs.newStore = make(map[string]*portRollupTracker)
+	prs.newStore = make(map[string][]uint16)
+}
+
+// GetCurrentStoreSize get number of tracked port counters in current store
+func (prs *EndpointPairPortRollupStore) GetCurrentStoreSize() int {
+	prs.mu.Lock()
+	defer prs.mu.Unlock()
+
+	return len(prs.curStore)
 }
 
-func buildStoreKey(sourceAddr []byte, destAddr []byte) string {
-	return string(sourceAddr) + "|" + string(destAddr)
+// GetNewStoreSize get number of tracked port counters in new store
+func (prs *EndpointPairPortRollupStore) GetNewStoreSize() int {
+	prs.mu.Lock()
+	defer prs.mu.Unlock()
+
+	return len(prs.newStore)
+}
+
+func buildStoreKey(sourceAddr []byte, destAddr []byte, endpointT endpointType, port uint16) string {
+	var portPart1, portPart2 = uint8(port >> 8), uint8(port & 0xff)
+	return string(sourceAddr) + string(destAddr) + string([]byte{byte(endpointT)}) + string([]byte{portPart1, portPart2})
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
We could implement a String() method in the endpoint enum instead of having to rely on `string([]byte{byte(endpointT)})`

Reply:
I would prefer keeping this inline:
- Since we might refactor this in the future
- String() is usually made to return something readable and readable string form, but `string([]byte{byte(endpointT)})` here is not readable, it's just `[]byte` casted to string.